options/posix/include/arpa/nameser.h

```diff
@@ -0,0 +1,209 @@
+#ifndef _ARPA_NAMESER_H
```

Comment:
The entire ``resolv`` library is actually defined as a part of BSD, and not POSIX. Should we maybe move it to its own option?

Reply:
We could, but so far BSD extensions have fallen under posix. For me, this is fine, but moving is always possible.

Reply:
I've moved this to a BSD option as discussed on Discord.